### PR TITLE
v70.1.0

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by romoore** [comment]
Looks like this is WIP. Removing reviewers for now.

**Comment by romoore** [comment]
This is missing a description, which I expect would be the release notes. @jverkoey would you please add them so we can review?

**Comment by jverkoey** [comment]
We don't add descriptions to the PR because it is a merge commit in the end.

**Comment by romoore** [comment]
I thought all of our PRs are squashed-and-merged? Aren't they all merge commits?

**Comment by jverkoey** [comment]
Releases are not squashed, they are merged in to both stable and develop.

File: catalog/MDCCatalog/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>70.0.0</string>
+	<string>70.1.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
```
Comment (romoore) [inline catalog/MDCCatalog/Info.plist:21]: Should the CFBundleVersion value be correct as well?

**Review by romoore — APPROVED** [review]
LGTM

File: catalog/MDCCatalog/Info.plist (hunk 1 above)
Comment (jverkoey) [reply catalog/MDCCatalog/Info.plist:21]: I'm not sure - it looks like we haven't been updating it. Should we?

**Pull request merged**